src/test/java/api/support/APITests.java

```diff
@@ -262,8 +251,13 @@
   protected final AutomatedPatronBlocksFixture automatedPatronBlocksFixture =
     new AutomatedPatronBlocksFixture();
 
-  protected final TenantActivationFixture tenantActivationFixture =
-    new TenantActivationFixture(restAssuredClient);
+  protected final TenantActivationFixture tenantActivationFixture = new TenantActivationFixture(restAssuredClient);
+  @Delegate
+  // The @Delegate annotation will instruct lombok to auto generate delegating methods
+  // in this class for all public methods of the PoliciesActivationFixture class
+  protected final PoliciesActivationFixture policiesActivation = new PoliciesActivationFixture();
```

Comment:
Was this done so that the clients to these methods are unaware that they have been moved?

Reply:
Yes, the clients are still able to reference methods from `APITests` class, but calls delegated to the fixture.

Reply:
Thank you for explaining this